### DISCUSSION TITLE
Edit Layout of Soft Keys

### DISCUSSION THREAD

**Discussion by theinfinitycorp** [rafaelvcaetano/melonDS-android#113] [Ideas]
This app has been running excellent on my device.
 I'm using this on a OnePlus 7 Pro. 
This is not a bug , just a feature suggestion! 
The controller keys are at the very bottom of the display . 

I wish that  the layout could be moved so that  the controls are  slightly above the bottom of the screen. It's kinda hard to press the keys being at the very bottom of the display.

**Comment by rafaelvcaetano** [comment]
Hi! Thanks for your feedback. Don't worry, that feature is in development. It won't come in the next release, but hopefully it doesn't take much longer.

**Comment by theinfinitycorp** [comment]
> Hi! Thanks for your feedback. Don't worry, that feature is in development. It won't come in the next release, but hopefully it doesn't take much longer.

Thanks a lot. This has been a awesome experience so far!

**Comment by aimdloop** [comment]
Editing layouts are possible rn in latest updates.
